static/incarcerated_case/uqhp_eligible/client_request.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
it's fine if you want to put subdirectories for "static/uqhp/incarceration" and "static/haven/incarceration" with multiple directories under each, though it might be overkill.